### PR TITLE
unit test exercises

### PR DESCRIPTION
**Pull request by mattz-taulia**

### CODE REVIEW

File: init/src/main/groovy/com/taulia/boundaries/Transmission.groovy [hunk 1]
```diff
@@ -2,9 +2,22 @@ package com.taulia.boundaries
...
     this.moveable = moveable
   }
+
+  void shift(Gear gear) {
+    if (canShiftToGear(gear)) {
+      this.gear = gear
+    }
+  }
+
+  private boolean canShiftToGear(Gear gear) {
```
Comment (brentfisher-taulia) [inline init/src/main/groovy/com/taulia/boundaries/Transmission.groovy:19]: I like this, pulling it out makes it more testable, and the intent is clearer too.  Great Job.

**Comment by brentfisher-taulia** [comment]
Looks great.  Might you be interested in completing the TimeTravellerTest?

**Comment by mattz-taulia** [comment]
What's the `TimeTravellerTest`? Is that in this project? (I was working on a temporary laptop at the time so I don't have it in front of me.)

**Comment by brentfisher-taulia** [comment]
https://github.com/brentfisher-taulia/boundaries/blob/master/init/src/test/groovy/com/taulia/boundaries/TimeTravellerTest.groovy

**Comment by brentfisher-taulia** [comment]
If you feel like it.  It helps to exercise the [T] in C O R R E C T. :)

**Pull request closed without merging**